### ISSUE TITLE
Wrap embedded and iFrame elements in JATS tags

### ISSUE THREAD

**Issue by gabestein** [pubpub/pubpub#1696]
Ensure that the assets.pubpub.org xlinks for non-images (such as mp3s) are included- In the older examples we looked at there was just an empty “boxed-text” tag where the audio clips are supposed to be.In the new sample the empty boxed-text is gone but there is no xlink for the non-images. Instead the embedded audio is expressed as:

`<p>[audio element]</p>`

And the iframe:

`<p>[iframe element]</p>`

Specifically, Karen was hoping to see something like this in the article-meta section (to instruct the web crawler what to harvest):
`<self-uri xlink:href=https://hdsr.mitpress.mit.edu/pub/xcq8a1v1 />`

And something like this in the body of the text (to use to check we have everything in the WARC file or to help us if we need to use the XML for rendition):            
```
<fig> <code ext-link=https://mit-harvard.netlify.com/#viz3 /></fig>
<fig><media mimetype="audio" mime-subtype="mp3" xlink:href=https://assets.pubpub.org/1ub4ka3t/21561644726227.mp3 /> </fig>
```

**Comment by gabestein** [comment]
I want to get more clarification from Portico about what they want here, and if there's a generic version that can suffice that doesn't force us to map HTML -> JATS in weird ways.

**Comment by idreyn** [comment]
Sounds good. I am having no luck wrapping JATS XML in a `RawBlock` in order to pass it directly into the output. We may need to hack together our own thing here to route around Pandoc. May be a question for jgm too.

**Comment by gabestein** [comment]
Yeah...how are we creating the [audio element] fill-ins right now?
